api/services/Discord.js

```diff
@@ -20,41 +84,35 @@ exports.getAccessToken = async function (code) {
 };
 
 exports.getCurrentUser = async function (token) {
+  const url = 'https://discordapp.com/api/users/@me';
+  const auth = 'Bearer ' + token;
+  const headers = {
+    "Authorization": auth,
+    "Content-Type": "application/x-www-form-urlencoded" 
+  };
   try {
-    const url = 'https://discordapp.com/api/users/@me';
-    const auth = 'Bearer ' + token;
-    const currentUser = await request.get({
-      url: url,
-      json: true,
-      headers: {
-        "Authorization": auth,
-        "Content-Type": "application/x-www-form-urlencoded",
-      }
-    });
-    return currentUser;
+    const currentUser = await makeRequest('GET', url, undefined, headers);
+    return currentUser.id;
```

Comment:
Is there any particular reason why you want to return `currentUser.id` instead `currentUser`? It'd fail to write details here:
https://github.com/robdy/flairhq/blob/f0177126d0281b50bc10354f9335a79120ecaf2d/api/controllers/AuthController.js#L111

Reply:
Reverted, sorry about that. I had another error that I was dealing with and I thought it was because of the currentUser, but I didn't really understand what I was working with at that point in time.